lib/kramdown_rpf/rpf.rb

```diff
@@ -3,25 +3,61 @@ module Plugin
     module Kramdown
       YAML_FRONT_MATTER_REGEXP = /\n\s*---\n(.*?)---(.*)/m
       KRAMDOWN_OPTIONS = {
-        coderay_css: :class, coderay_line_numbers: nil, parse_block_html: true, input: 'KramdownRPF'
+        input:              'KramdownRPF',
+        parse_block_html:   true,
+        syntax_highlighter: nil,
       }.freeze
 
       def self.convert_challenge_to_html(challenge)
-        ::Kramdown::Document.new(
-          "<div class=\"challenge\">\n#{challenge}</div>",
-          KRAMDOWN_OPTIONS
-        ).to_html
+        ::Kramdown::Document.new("<div class=\"challenge\">\n#{challenge}</div>", KRAMDOWN_OPTIONS).to_html
+      end
+
+      def self.convert_code_to_html(code_block)
+        code_block =~ YAML_FRONT_MATTER_REGEXP
+        meta       =  YAML.safe_load(Regexp.last_match(1))
+
+        language          = meta['language']
+        filename          = meta['filename'] or nil
```

Comment:
Sorry - should have spotted this before - I don't think `or` is good practice https://stackoverflow.com/questions/2083112/difference-between-or-and-in-ruby

Could you swap for `||`

Reply:
UGH. Yeah you're right. I've been writing loads of Lua recently, which doesn't have `||`, so I automatically slipped into it! Will replace. Interesting that Rubocop didn't catch it. I must not have it set up right.